### ISSUE TITLE
[BUG] batch进行generate时，Qwen的tokenizer会设置为right-padding，并且无法修改成left-padding。

### ISSUE THREAD

**Issue by zhangbin1997** [QwenLM/Qwen-7B#145]
### 是否已有关于该错误的issue？ | Is there an existing issue for this?

- [X] 我已经搜索过已有的issues | I have searched the existing issues

### 当前行为 | Current Behavior

![image](https://github.com/QwenLM/Qwen-7B/assets/64835626/6904d1b4-25c7-49dd-94cd-986eb1af9f03)
如图所示，模型会一直报right-padding的警告，但是我已经在tokenizer初始化的时候将 padding_side='left' 设置上了，除了Qwen的模型外，其他任何开源模型都不会存在这个问题，我怀疑可能是huggingface上的Qwen tokenizer 代码有bug，希望作者check一下。right-padding所导致的生成结果如下图所示。会生成非常长且冗余的！！！！
![image](https://github.com/QwenLM/Qwen-7B/assets/64835626/718eaf00-1483-46ae-8ef3-880a5aad15cc)

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```

### 备注 | Anything else?

另外希望作者提供一个内部的Qwen交流群，最好是阿里钉的交流群，谢谢^-^

**Comment by jklj077** [comment]
您好，感谢支持，目前我们没有复现出该问题。由于上周末重构过tokenizer，请先更新下代码试试？

![image](https://github.com/QwenLM/Qwen-7B/assets/17811943/9c9e9252-d587-41e7-bc21-96d74990ea8b)

**Comment by jklj077** [comment]
由于问题未复现且用户未回复，本issue视为已解决。如果问题还存在请重开该issue，感谢！
